docs/source/form/select/components/select.md

```diff
@@ -72,7 +72,7 @@ Allow new items to be created if not found. **Default:** `false`.
 
 ### `allowSelectAll?: boolean`
 
-Adds a `Select all` option ( when `isMulti` is `true`).
+Adds a `Select all` option ( when `isMulti` is `true`). Note: allowSelectAll does not support use of the `loadOptions` prop.
```

Comment:
nit: suggesting rephrase to:

Note: allowSelectAll is ignored when `loadOptions` is defined.

Reply:
done